package.json

```diff
@@ -14,7 +14,6 @@
     "lint": "esw src test"
   },
   "dependencies": {
-    "@syncano/cli": "^0.9.3",
```

Comment:
why remove the `@syncano/cli` package?

Reply:
Had a previous understanding that it should be globally installed, but this is fixed now